From 300939fd6ddb6ea3e5a1621727236bb2b6d54836 Mon Sep 17 00:00:00 2001
From: avishad verma <avishad.verma@infineon.com>
Date: Fri, 25 Nov 2022 03:07:16 -0600
Subject: [PATCH 08/82] brcmfmac: Update D2H Validation Done Timeout.

Increasing the timeout to accomadate validation of Firmware with bigger
size and RSA length.

Upstream-Status: Inappropriate [Infineon downstream https://github.com/Infineon/ifx-wireless-drivers b22c204af79f3754cfae6764ae74a936f986f1ce brcmfmac: Update D2H Validation Done Timeout.
30395dee909eba36a8e9eaf1a8a1058a18d84444 brcmfmac: Update D2H Validation Done Timeout.]
Fixes SWWLAN-145154
Signed-off-by: avishad verma <avishad.verma@infineon.com>
---
 drivers/net/wireless/broadcom/brcm80211/brcmfmac/chip.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/chip.c b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/chip.c
index 24092ff248e57..35b58a54061da 100644
--- a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/chip.c
+++ b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/chip.c
@@ -240,7 +240,8 @@ struct sbsocramregs {
 #define BRCMF_BLHS_POLL_INTERVAL			10	/* msec */
 #define BRCMF_BLHS_D2H_READY_TIMEOUT			100	/* msec */
 #define BRCMF_BLHS_D2H_TRXHDR_PARSE_DONE_TIMEOUT	50	/* msec */
-#define BRCMF_BLHS_D2H_VALDN_DONE_TIMEOUT		250	/* msec */
+
+#define BRCMF_BLHS_D2H_VALDN_DONE_TIMEOUT		450	/* msec */
 
 /* Bootloader handshake flags - dongle to host */
 #define BRCMF_BLHS_D2H_START			BIT(0)
-- 
2.51.0

